src/controllers/tripsController.js

```diff
@@ -516,4 +516,24 @@ export default class requestController {
       return Response.errorResponse(res, 500, res.__('server error'));
     }
   }
+
+  /**
+   * @description this function finds and displays most travelled destinations
+   * @param  {object} req
+   * @param  {object} res
+   * @return {object} most travelled destinations
+   */
+  static async mostTravelledDestinations(req, res) {
+    try {
+      const counting = await db.Request.findAll({
```

Comment:
Should we get all trips or all approved trips?

Reply:
it would be better to base on approved trips... thanks for that interesting suggestion.